.gitattributes

```diff
@@ -1,2 +1,12 @@
 # Normalize EOL for all files that Git considers text files.
 * text=auto eol=lf
+
+# Exclude everything outside /addons/block_code
+/.*                       export-ignore
+/LICENSE                  export-ignore
+/README.md                export-ignore
+/addons/plugin_refresher  export-ignore
+/docs                     export-ignore
+/icon.svg*                export-ignore
+/project.godot            export-ignore
+/test_game                export-ignore
```

Comment:
I re-read the documentation and I think this can be made simpler actually. 1 moment.

Reply:
Compared to your original approach there is just 1 line change:

```diff
-# Only include the addons folder when downloading from the Asset Library.
+# Only include the block coding addon when downloading from the Asset Library.
+# Exclude non-addon files and the plugin_refresher addon used during
+# development.
 /**                    export-ignore
+/addons               !export-ignore
 /addons/block_code    !export-ignore
 /addons/block_code/** !export-ignore
```

Reply:
Yes, go ahead! Thanks for letting me know how to test this locally.